stable/traefik/values.yaml

```diff
@@ -58,6 +60,14 @@ service:
   #   key: value
   # labels:
   #   key: value
+  ## Further config for service of type NodePort
+  ## Configure ports in allowable range (eg. 30000 - 32767 on minikube)
+  ## nodePorts:
+  ##   http: 31000
+  ##   https: 32000
```

Comment:
Nit: Can I suggest port numbers that bear some kind of resemblance to the privileged ports they map to within the Traefik container? 30080 and 30443 sound reasonable.

Reply:
Yeah that makes sense. Will update